apps/api/src/app/events/services/storage-helper-service/storage-helper.service.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { expect } from 'chai';
+import { S3Client, PutObjectCommand, GetObjectCommand, DeleteObjectCommand } from '@aws-sdk/client-s3';
+import { mockClient } from 'aws-sdk-client-mock';
+import { S3StorageService } from '../../../shared/services/storage/storage.service';
+import { StorageHelperService } from './storage-helper.service';
+
+const s3Mock = mockClient(S3Client);
+
+describe('Storage-Helper service', function () {
+  describe('S3', function () {
```

Comment:
I think we should also test the Azure and GCP bucket implementation besides AWS one.

Reply:
@p-fernandez Can you say how to do that? `LOCALSTACK` works for them?

Reply:
Unfortunately Localstack is only for AWS storage. For GCP and Azure storage you would need to mock the dependencies using Sinon.